paper/paper.md

```diff
@@ -1,46 +1,103 @@
+
+```python
+from splot.esda import plot_local_autocorrelation
+plot_local_autocorrelation(moran_loc, gdf, 'Crime')
+plt.show()
+```
+
+
+[![Fig. 1: Local Spatial Autocorrelation](figs/local_autocorrelation.png)](https://github.com/pysal/splot/blob/master/notebooks/esda_morans_viz.ipynb)
+
+The user can now further visually assess whether there is dependency between high crime rates (fig. 2, rgb variable) and high income in this neighborhood (fig. 2, alpha variable). Darker shades of the colormap correspond to higher crime and income values, displayed through a static Value-by-Alpha Choropleth using `splot.mapping.vba_choropleth`.
+
 
+```python
+fig = plt.figure(figsize=(10,10))
+ax = fig.add_subplot(111)
+vba_choropleth(x, y, gdf,
+               alpha_mapclassify=dict(classifier='quantiles', k=5,
+               rgb_mapclassify=dict(classifier='quantiles', k=5,
+               cmap='Blues',
+               legend=True, divergent=True, ax=ax)
+plt.show()
+```
 
-# Citations
 
-Citations to entries in paper.bib should be in
-[rMarkdown](http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html)
-format.
+[![Fig. 2: Value-by-alpha mapping](figs/vba_choropleth.png)](https://github.com/pysal/splot/blob/master/notebooks/mapping_vba.ipynb)
 
-# Figures
+Ultimately, the `splot` package is designed to facilitate the creation of both static plots ready for publication, and interactive visualizations for quick iteration and spatial data exploration. Although most of `splot` is currently implemented with a `matplotlib` backend, `splot` is framework independent. In that sense, `splot` offers a "grammar" of views that are important and useful in spatial analyses and geographic data science. The `splot` package is not restricted or limited to the current `matplotlib` implementation and can be advanced by integrating emerging or succeeding interactive visualization toolkits, such as `altair` or `bokeh`.
 
-Figures can be included like this: ![Example figure.](figure.png)
+In conclusion, `splot` tightly connects visual analytics with statistical analysis and facilitates the integration of spatial analytics into more general Python workflows through it's compatibility with integrated code-based environments like Jupyter. From spatial autocorrelation analysis to value by alpha choropleths, `splot` is designed as a grammar of views that can be applied to a multitude of spatial analyses workflows. As `splot` developers, we strive to expand `splot`'s grammar of views through new functionality (e.g. in flow mapping methods), as well as provide different backend implementations, including interactive backends, such as `bokeh`, in the future.
```

Comment:
> ... can be applied to a **multitude** of spatial analyses workflows.

Already used "multitude" earlier int he paper. Maybe swap out for:
 * "... a variety of spatial analyses workflows."
 * "... a diverse collection of spatial analyses workflows."
 * etc.

Reply:
I missed the "*spatial analyses workflows*" here... Should be "*spatial analysis workflows.*"